frontend/packages/schema-editor/src/components/NodePanel/NodePanel.tsx

```diff
@@ -38,9 +38,11 @@ const BackButton = () => {
   };
 
   return (
-    <Link as='button' onClick={navigateToDatamodelRoot} className={classes.backButton}>
-      <ArrowLeftIcon />
-      {t('schema_editor.back_to_datamodel')}
+    <Link asChild onClick={navigateToDatamodelRoot} className={classes.backButton}>
+      <button>
+        <ArrowLeftIcon />
+        {t('schema_editor.back_to_datamodel')}
+      </button>
     </Link>
```

Comment:
Her ville jeg heller satt `onClick` på `button`-elementet. Hadde `navigateToDatamodelRoot` tatt imot eventet som et parameter, ville vi fått en typeadvarsel her. Litt av poenget med denne nye syntaksen er også å få bedre typevalidering.
```suggestion
    <Link asChild className={classes.backButton}>
      <button onClick={navigateToDatamodelRoot}>
        <ArrowLeftIcon />
        {t('schema_editor.back_to_datamodel')}
      </button>
    </Link>
```

Reply:
Det gir mening :)